src/main/java/org/broadinstitute/hellbender/tools/walkers/concordance/ConcordanceSummaryRecord.java

```diff
@@ -0,0 +1,106 @@
+package org.broadinstitute.hellbender.tools.walkers.concordance;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.utils.tsv.TableReader;
+import org.broadinstitute.hellbender.utils.tsv.TableWriter;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by tsato on 2/8/17.
+ */
+
+public class ConcordanceSummaryRecord {
+    private static final String VARIANT_TYPE_COLUMN_NAME = "type";
+    private static final String TRUE_POSITIVE_COLUMN_NAME = "true-positive";
+    private static final String FALSE_POSITIVE_COLUMN_NAME = "false-positive";
+    private static final String FALSE_NEGATIVE_COLUMN_NAME = "false-negative";
+    private static final String SENSITIVITY_COLUMN_NAME = "sensitivity";
+    private static final String PRECISION_COLUMN_NAME = "precision";
+    private static final String[] SUMMARY_TABLE_COLUMN_HEADER =
+            {VARIANT_TYPE_COLUMN_NAME, TRUE_POSITIVE_COLUMN_NAME, FALSE_POSITIVE_COLUMN_NAME,
+                    FALSE_NEGATIVE_COLUMN_NAME, SENSITIVITY_COLUMN_NAME, PRECISION_COLUMN_NAME};
+
+    final VariantContext.Type type;
+    final long truePositives;
+    final long falsePositives;
+    final long falseNegatives;
+
+    public ConcordanceSummaryRecord(final VariantContext.Type type, final long truePositives, final long falsePositives, final long falseNegatives){
+        this.type = type;
+        this.truePositives = truePositives;
+        this.falsePositives = falsePositives;
+        this.falseNegatives = falseNegatives;
+    }
+
+    public VariantContext.Type getVariantType(){
```

Comment:
If you like one-liners, trivial getters are one place where GATK style permits them.